### ISSUE TITLE
Is this abandeoned?

### ISSUE THREAD

**Issue by dilirity** [Ceddini/CoreKeeperMapTool#40]
Asking, since it's a very nice tool. Would love to know if it's going to get more support.

**Comment by Micke90s** [comment]
@dilirity I don’t think that the project is completely abandoned yet. It seems that @Ceddini is working on [Item Signs Mod]( https://mod.io/g/corekeeper/m/item-signs) at the moment.

**Comment by Ceddini** [comment]
Mostly working on the actual game at the moment. Need to check if I'm allowed to keep working on it but then I can probably update it to 1.0
